### DISCUSSION TITLE
Vertical split

### DISCUSSION THREAD

**Discussion by schultzter** [nvim-orgmode/orgmode#239] [Ideas]
Is there a way to get the Agenda and org files to split vertically instead of horizontally in the Neovim editor window?

**Comment by TravonteD** [comment]
Not currently, this is something that we can add

**Comment by kristijanhusak** [comment]
This is now possible via [win_split_mode](https://github.com/nvim-orgmode/orgmode/blob/master/DOCS.md#win_split_mode) option.
